Library/Homebrew/items.sh

```diff
@@ -17,6 +17,7 @@ homebrew-items() {
 
   # HOMEBREW_REPOSITORY is set by brew.sh
   # shellcheck disable=SC2154
+  [[ -d "${HOMEBREW_REPOSITORY}/Library/Taps" ]] || return
```

Comment:
I wonder if we can use `Homebrew::API::Formula.all_formulae.keys`. Alternatively, could run a hack and extract through shell script (grep, sed, etc.) if cache file exists.